### ISSUE TITLE
How to add description or other information to Inheritors

### ISSUE THREAD

**Issue by dfroze** [Kotlin/dokka#2505]
Hi,

So I have multiple classes that has as interface "Builder", as seen in the image.
![2022-05-16_17-06](https://user-images.githubusercontent.com/7603482/168611868-3db8946b-992e-438a-8297-18f3d6718888.png)

I've added description for each class, but nothing shows when it is generated.
```/**
     * Builder to generate [Config]
     */
    class Builder(ctx: Context): com.example.Builder {
}`
```
Am I missing something or is there no solution for this?

**Comment by IgnatBeresnev** [comment]
Hi! 

I take it you expected documentation to be shown next to each individual inheritor? If so, it's not implemented to be this way, but it may be a good idea to add some short description as already done for functions.

Otherwise you may find your builder documentation if you click one of the links, it'll redirect you to that Builder's page and you will see the docs there

I'll modify the issue to be an enhancement if you don't mind :)

**Comment by dfroze** [comment]
Yes, a short description for each one would be great. I find it hard to navigate from this position without knowing anything about each inheritor (destination). Maybe also display the class name of each inheritor instead of 6 time just "Builder" .
